### ISSUE TITLE
Documentation for FOIA request intake API

### ISSUE THREAD

**Issue by konklone** [18F/foia-hub#38]
@khandelwal, hoping you can take a stab at a Markdown doc with documentation for our request intake API, as we talked about last week.

I just laid out the state of things in https://github.com/18F/foia/issues/25, and some ideas about what to do next, so this should be done in parallel to updating fields, which I'll add in a separate ticket.
